packages/react-components/react-combobox/src/components/Listbox/useListbox.ts

```diff
@@ -90,6 +89,41 @@ export const useListbox_unstable = (props: ListboxProps, ref: React.Ref<HTMLElem
     }
   };
 
+  const onFocus = (_event: React.FocusEvent<HTMLElement>) => {
+    if (hasParentActiveDescendantContext) {
+      return;
+    }
+
+    // set initial active descendent upon focus, in the absence of a parent context controlling the state
+    const activeDescendent = activeDescendantController.active();
+    if (activeDescendent) {
+      activeDescendantController.focus(activeDescendent);
+      return;
+    }
```

Comment:
```suggestion
```

I don't think this is necessary - if there is an active element, there should be no need to focus it again

Reply:
Replaced this with memorization logic and added a unit test :)